pincer/objects/guild/channel.py

```diff
@@ -645,21 +613,377 @@ async def create_invite(
+        auto_archive_duration: Optional[:class:`int`]
+            The duration in minutes to automatically archive the thread after
+            recent activity, can be set to: ``60``, ``1440``, ``4320``, ``10080``.
+        rate_limit_per_user: Optional[:class:`int`]
+            Amount of seconds a user has to wait before sending another message
+            (0-21600)
+        reason: Optional[:class:`str`]
+            The reason of the thread creation.
+
+        The 3 day and 7 day archive durations require the server to be boosted.
+        The guild features will indicate if a server is able to use those settings.
+
+        Returns
+        -------
+        :class:`~pincer.objects.channel.Channel`
+            The created thread.
+        """
+        return Channel.from_dict(
+            construct_client_dict(
+                self._client,
+                await self._http.post(
+                    f"channels/{self.id}/messages/{message.id}/threads",
+                    headers=remove_none({"X-Audit-Log-Reason": reason}),
+                    data={
+                        "name": name,
+                        "auto_archive_duration": auto_archive_duration,
+                        "rate_limit_per_user": rate_limit_per_user,
+                    },
+                ),
+            )
+        )
+
+    async def start_thread(
+        self,
+        name: str = None,
+        auto_archive_duration: Optional[int] = None,
+        type_: Optional[ChannelType] = None,
+        invitable: Optional[bool] = None,
+        rate_limit_per_user: Optional[int] = None,
+        reason: Optional[str] = None,
+    ) -> Channel:
+        """
+        Creates a new thread that is not connected to an existing message.
+        The created thread defaults to a ``GUILD_PRIVATE_THREAD``*.
+        Returns a Channel on success.
+
+        Parameters
+        ----------
+        name: Optional[:class:`str`]
+            The name of the thread. 1-100 characters.
+        auto_archive_duration**: Optional[:class:`int`]
+            The duration in minutes to automatically archive the thread after
+            recent activity, can be set to: ``60``, ``1440``, ``4320``, ``10080``.
+        type_: Optional[:class:`~pincer.objects.channel.ChannelType`]
+            The type of thread to create.
+        invitable: Optional[:class:`bool`]
+            Whether non-moderators can add other non-moderators to a thread;
+            only available when creating a private thread.
+        rate_limit_per_user: Optional[:class:`int`]
+            Amount of seconds a user has to wait before sending another message.
+            (0-21600)
+        reason: Optional[:class:`str`]
+            The reason of the thread creation.
+
+        \\*: Creating a private thread requires the server to be boosted.
+        The guild features will indicate if that is possible for the guild.
+
+        \\*\\*: The 3 day and 7 day archive durations require the server to be boosted.
```

Comment:
```suggestion
        \\*args: Creating a private thread requires the server to be boosted.
        The guild features will indicate if that is possible for the guild.

        \\*\\*kwargs: The 3 day and 7 day archive durations require the server to be boosted.
```

Reply:
bruh did you even bother to check what this was for :skull:

Reply:
no :skull: